[DOCS-10961] Add info on editing ootb detection rules (#29502)
DataDog/documentation: content/en/security/cloud_siem/detection_rules/_index.md

## Overview

> Cloud SIEM detection rules analyze logs and audit events and generate security signals when a threat is detected. You can use [out-of-the-box detections rules](#out-of-the-box-detection-rules) or [create custom detection rules](#custom-detection-rules). This document walks you through how to create a custom detection rule.

Author: ```suggestion
Cloud SIEM detection rules analyze logs and audit events, and generate security signals when a threat is detected. You can use [out-of-the-box detections rules](#out-of-the-box-detection-rules) or [create custom detection rules](#custom-detection-rules). This document walks you through how to create a custom detection rule.
```

> Cloud SIEM detection rules analyze logs and audit events, and generate security signals when a threat is detected. You can use [out-of-the-box detections rules](#out-of-the-box-detection-rules) or [create custom detection rules](#custom-detection-rules). This document walks you through how to create a custom detection rule.

Reviewer: ```suggestion
Cloud SIEM detection rules analyze logs and audit events to generate security signals when threats are detected. You can use [out-of-the-box detections rules](#out-of-the-box-detection-rules) or [create custom detection rules](#custom-detection-rules). This document walks you through how to create a custom detection rule.
```

### Out-of-the-box detection rules

> When Cloud SIEM has been setup, [OOTB detection rules][6] automatically start analyzing your logs and audit events. You can edit OOTB detection rules and:

Reviewer: ```suggestion
After you set up Cloud SIEM, [OOTB detection rules][6] automatically begin analyzing your logs and audit events. You can edit OOTB detection rules and:
```
